source/.ebextensions/fix-npm.config

```diff
@@ -87,11 +87,18 @@ files:
       # need to be rebuilt. When the Node version changes, the configdeploy script
       # will rebuild.
       #
+      #
       # Note that this *overwrites* Elastic Beanstalk's default 50npm.sh script.
       
       . /opt/elasticbeanstalk/env.vars
 
-      cd $EB_APP_STAGING_DIR && npm install --production
+      cd $EB_APP_STAGING_DIR            
+
+      if [ -d node_modules ]; then # If there is an already existent node_modules in the deployed app (built by the AWS CodeBuild) 
```

Comment:
Mixmax no longer uses this script [1], so I can't easily test, but I'm pretty sure that this will always be `true`, since `node_modules` is [cached](https://github.com/yanisIk/eb-fix-npm/blob/b19267aa1ae3b74f3bb90580f999fccddd3a2468/source/.ebextensions/fix-npm.config#L75). Moreover when _not_ shipping `node_modules` to the EB environment, on the first deploy this cache directory would be empty, so the modules wouldn't get installed.

[1] in favor of [`eb-disable-npm`](https://github.com/mixmaxhq/eb-disable-npm), which—not to dissuade you from this—you might consider too, since it sounds like your setup works like ours as far as shipping `node_modules` to the EB environments. Though that does have its [downside](https://github.com/mixmaxhq/eb-disable-npm#warning).